lib/meilisearch/index.rb

```diff
@@ -225,6 +225,7 @@ def delete_all_documents!
     #   attributes_to_search_on - Customize attributes to search on at search time.
     def search(query, options = {})
       attributes = { q: query.to_s }.merge(options.compact)
+      attributes.delete :matching_strategy
```

Comment:
I'm not sure if I miss something but I was not expecting this here...

Reply:
Oh lmao my bad. This is why you don't do git add . xD
That was just me testing if the specs break  when I remove the feature. I'll remove it.

Reply:
It should be removed now